### PR TITLE
Reusable Navigation Arrows Ready for Use

### PR DESCRIPTION
**Pull request by ericliddle**
Created left and right navigation components that are ready for use.

### CODE REVIEW

File: js/components/LeftNavArrow/styles.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { StyleSheet } from 'react-native';
+
+export const styles = StyleSheet.create({
+  container: {
+
```
Comment (10thfloor) [inline js/components/LeftNavArrow/styles.js]: Fix extra whitespace.

File: js/components/LeftNavArrow/styles.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    position: 'absolute',
+    left: 10,
+    top: '50%'
+  },
+  arrowContainer: {
```
Comment (10thfloor) [inline js/components/LeftNavArrow/styles.js]: Remove empty style rule.

File: js/components/RightNavArrow/styles.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { StyleSheet } from 'react-native';
+
+export const styles = StyleSheet.create({
+  container: {
```
Comment (10thfloor) [inline js/components/RightNavArrow/styles.js]: Fix whitespace.

**Review by 10thfloor — CHANGES_REQUESTED** [review]

File: js/components/LeftNavArrow/styles.js (hunk 1 above)
Comment (ericliddle) [reply js/components/LeftNavArrow/styles.js]: Fixed white space fro Left and Right Nav Arrow

File: js/components/RightNavArrow/styles.js (hunk 3 above)
Comment (ericliddle) [reply js/components/RightNavArrow/styles.js]: Fixed white space fro Left and Right Nav Arrow

File: js/components/LeftNavArrow/styles.js (hunk 2 above)
Comment (ericliddle) [reply js/components/LeftNavArrow/styles.js]: Removed empty rule from Styles and Left and Right Nav file.

**Review by 10thfloor — CHANGES_REQUESTED** [review]
Fix conflicts, please!

**Review by 10thfloor — APPROVED** [review]

**Pull request merged**